controllers/drplacementcontrol_controller.go

```diff
@@ -710,7 +714,17 @@ func (r *DRPlacementControlReconciler) Reconcile(ctx context.Context, req ctrl.R
 
 	ensureDRPCConditionsInited(&drpc.Status.Conditions, drpc.Generation, "Initialization")
 
-	placementObj, err := getPlacementOrPlacementRule(ctx, r.Client, drpc, logger)
+	_, ramenConfig, err := ConfigMapGet(ctx, r.APIReader)
+	if err != nil {
+		statusErr := r.updateDRPCStatus(ctx, drpc, placementObj, logger)
```

Comment:
Shouldn't we be using recordFailure here?

Reply:
Done.